electron-app/src/server/websites/telegram/telegram.service.ts

```diff
@@ -32,9 +32,9 @@ import { GenericAccountProp } from '../generic/generic-account-props.enum';
 import { LoginResponse } from '../interfaces/login-response.interface';
 import { ScalingOptions } from '../interfaces/scaling-options.interface';
 import { Website } from '../website.base';
+import { TelegramDescription } from './telegram-description.parser';
 import { TelegramStorage } from './telegram.storage';
 import _ = require('lodash');
-import { TelegramDescription } from './telegram-description.parser';
 
 @Injectable()
 export class Telegram extends Website {
```

Comment:
q: I am not exactly sure how this fixes anything.
Can you elaborate?

Reply:
If you mean moving the import around - its the vscode's sort import thingy, i just forgot to turn it off for this project

If you mean the whole diff, then the main point of fixing is using current date as a offset instead of 0 date (also some small fixes such as saving folders after each request to show atleast something to the user because loading channels can take long. And also some additional logging)